backend/dynamo-api/src/handlers/get-service-clients.ts

```diff
@@ -1,29 +1,21 @@
-import {APIGatewayProxyEvent, APIGatewayProxyResult} from 'aws-lambda';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
 import DynamoClient from "../client/DynamoClient";
 
-const tableName = process.env.TABLE;
-const client = new DynamoClient(tableName as string);
-
-
 export const getServiceClientsHandler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
     const serviceId = event.pathParameters?.serviceId;
     if (serviceId === undefined) {
         return noUserIdResponse;
     }
 
-    let response = {statusCode: 200, body: JSON.stringify(serviceId)};
-    await client.getClients(serviceId)
+    return new DynamoClient()
```

Comment:
It's considered best practice to instantiate as much as possible at the top level of the code so that it happens at init rather than invocation.  It may offend your sense of order and rightness but it was in the right place for a Lambda.

Reply:
It was so save lines of code but it makes sense to initialise things at init. Will change this back

Reply:
It offends _my_ sense of order and rightness 😆 .  Even more so in Java when you start constructing outside the constructor but I guess it's not as bad here in TS land.